corehq/apps/app_manager/_design/views/xforms_index/map.js

```diff
@@ -6,14 +6,19 @@ function (doc) {
             "unique_id": form.unique_id
         });
     }
-    if (doc.doc_type === 'Application' && !doc.copy_of) {
-        if (doc.user_registration) {
-            doEmit(doc.user_registration);
-        }
-        for (m in doc.modules) {
-            for (f in doc.modules[m].forms) {
-                doEmit(doc.modules[m].forms[f]);
-            }
+    if (doc.doc_type.indexOf(['Application', 'LinkedApplication']) === -1) {
```

Comment:
shouldn't this be reversed `['Application', 'LinkedApplication'].indexOf(doc.doc_type)`